sql/create_category_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE category (
+  categoryId INT PRIMARY KEY AUTO_INCREMENT,
```

Comment:
completed the code change  of using Integer instead of int and use id instead of category_id and subCategory_id .
Updated and used name instead of category_name and subCategory_name.